docs/mem_quick_start.ipynb

```diff
@@ -0,0 +1,517 @@
+    "\n",
+    "* `user` : Database user to use for built-in database authentication and login\n",
+    "* `password` : Database password to use for built-in database authentication and login.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note**: This tutorial demonstrates the async interface."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBEngine\n",
+    "\n",
+    "engine = await AlloyDBEngine.afrom_instance(\n",
+    "    project_id=PROJECT_ID,\n",
+    "    region=REGION,\n",
+    "    cluster=CLUSTER,\n",
+    "    instance=INSTANCE,\n",
+    "    database=DATABASE,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create an AlloyDBModelManager instance\n",
+    "The `AlloyDBModelManager` class allows the user to create a model, list a model, list all models and drop a model. By creating a model using this, the `AlloyDBEmbedding` class can be used to call that specific model to embed all the data and added into the vectorstore."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBModelManager\n",
+    "\n",
+    "model_manager = AlloyDBModelManager(engine)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "On creating the `AlloyDBModelManager` object, it will run a prerequisite check.\n",
+    "In case of an error make sure:\n",
+    "* extension is up to date : `google_ml_integration` extension is installed and the version is greater than 1.3\n",
+    "* db flag is set : `google_ml_integration.enable_model_support` is set to on."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### To list all models available\n",
+    "This list includes the two pre built models:\n",
+    "* `textembedding-gecko`\n",
+    "* `textembedding-gecko@001`\n",
+    "\n",
+    "And any other model you may have created\n",
+    "\n",
+    "The alist_models function returns `List[AlloyDBModel]`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(await model_manager.alist_models())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### To create a custom text embedding model\n",
+    "The acreate_model function has three required parameters:\n",
+    "* model_id: A unique ID for the model endpoint that you define.\n",
+    "* model_provider: The provider of the model endpoint (`google` for vertexAI and `custom` for custom  hosted models).\n",
+    "* model_type: The model type (set this value to `text_embedding` for text embedding model endpoints or `generic` for all other model endpoints).\n",
```

Comment:
model_qualitifed_name might be needed

Reply:
Have made the model_qualified_name a required parameter in both this notebook and in the code PR